internal/pkg/describe/worker_service.go

```diff
@@ -145,6 +156,7 @@ type workerSvcDesc struct {
 	Type           string               `json:"type"`
 	App            string               `json:"application"`
 	Configurations ecsConfigurations    `json:"configurations"`
+	Alarms         []string             `json:"rollbackAlarms,omitempty"`
```

Comment:
do we not want to include the "Rollback Alarms" in `HumanString` for worker services?

Reply:
🤦🏼‍♀️